data/scoreboard_config.py

```diff
@@ -97,6 +99,13 @@ def check_preferred_divisions(self):
       division = self.preferred_divisions
       self.preferred_divisions = [division]
 
+  def check_time_format(self):
+    if self.time_format.lower() == "12h":
+      self.time_format = "%-I"
+    elif self.time_format.lower() == "24h":
```

Comment:
Would probably be simpler to just check for 24 and set it, otherwise always fall back to 12

Reply:
Ya. That was just silly.